core/src/bson_type_info.rs

```diff
@@ -840,6 +891,58 @@ impl BsonTypeInfo {
         column_size: make_default_attr_func!(None),
         simple_type_info: SimpleTypeInfo::default(),
     };
+    // This is just here to support any tools that attempt to cast to NUMERIC.
+    pub const NUMERIC: BsonTypeInfo = BsonTypeInfo {
```

Comment:
[Note]
I have a feeling that things can still fail, because Numeric is for exact numeric value which means a fixed precision and scale.
We don't define any boundaries for it. So, similarly to what happened for VARCHAR, PowerBI might not like the unknown.

Reply:
That was only a problem because it was trying to find a type big enough to contain 4 for something like "TRUE". I'm not sure what to do here, we might want to just discuss it quickly when you are back